sdk/python/approzium/psycopg2/__init__.py

```diff
@@ -0,0 +1,4 @@
+from ._psycopg2_connect import connect, AUTH_REQ_MD5, AUTH_REQ_SASL
+
+
+__all__ = ['connect', 'AUTH_REQ_MD5', 'AUTH_REQ_SASL']
```

Comment:
using `__all__` is the preferred way to define the interface for submodule. before, we were exporting many private names.